packages/dm-core-plugins/src/attribute-selector/types.tsx

```diff
@@ -10,7 +10,10 @@ export type TAttributeSelectorItem = {
   label?: string
 }
 export type TItemData = TAttributeSelectorItem & {
+  key: string
```

Comment:
The`key` is a generic name, should it be renamed to viewId, itemId (or something like that)? Or, could the key be generated from the information that exists in the view config? Perhaps combine the name of the view config and the scope to get the unique key?